src/main/resources/templates/subscription.html

```diff
@@ -123,6 +123,7 @@ <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                                         <span data-bind="text: label"></span>
                                     </label>
                                 </div>
+
```

Comment:
You may revert all changes in this file.

Reply:
ok